instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes_test.go

```diff
@@ -40,3 +44,29 @@ func TestSystemAttribute(t *testing.T) {
 	attr := SystemAttr()
 	assert.Equal(t, semconv.RPCSystemKey.String("aws-api"), attr)
 }
+
+func TestDefaultAttributeBuilder_ShouldReturnNoAttributesOnNotSupportedService(t *testing.T) {
```

Comment:
We are not using `_` in the test names. Can you also make the test name shorter? E.g.



```suggestion
func TestDefaultAttributeBuilderNotSupportedService(t *testing.T) {
```